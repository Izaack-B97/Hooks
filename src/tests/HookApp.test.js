import { shallow } from "enzyme";
import { HookApp } from "../HookApp";

describe('Pruebas en <HookApp />' , () => {
    
    test('debe de reenderizar el hola mundo ', () => {
        
        const wrapper = shallow( <HookApp /> );

        expect( wrapper ).toMatchSnapshot();

    });
    
});