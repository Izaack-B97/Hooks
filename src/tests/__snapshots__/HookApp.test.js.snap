// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> debe de reenderizar el hola mundo  1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
